testdata/transition-worker-2.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+while (frankenphp_handle_request(function () {
+    echo "Hello from worker 2";
+    usleep(1000);
```

Comment:
Why the sleep?

Reply:
The sleep here should again help force potential race conditions to appear (changing the thread handler while a request is running). Would you be fine with the sleep if I added a comment?

Reply:
Yes ok got it. A comment would be nice indeed (same for other similar cases). This will prevent unwanted removed of these sleeps.